servers/src/main/java/tachyon/worker/block/BlockMetadataManager.java

```diff
@@ -166,7 +166,7 @@ public void cleanupSessionTempBlocks(long sessionId, List<Long> tempBlockIds) {
    * @return available bytes
    * @throws IllegalArgumentException when location does not belong to tiered storage
    */
-  public long getAvailableBytes(BlockStoreLocation location) {
+  public long getAvailableBytes(BlockStoreLocation location) throws IllegalArgumentException {
```

Comment:
my view is `IllegalArgumentException` is a subclass of `java.lang.RuntimeException`, which we don't usually put into method signature. Do you have a particular reason of doing this, just to match the javadoc which declares throwing this exception? (if it is for the latter case,  I suggest to update the javadoc instead)